### PR TITLE
Some clean up

### PR DESCRIPTION
**Pull request by danfishgold**
A couple of smallish changes before the interesting stuff

- Move some scss code around to be more representative of DOM structure
- Remove old colors
- Update TypeScript to version 4.4

### CODE REVIEW

File: src/components/Export.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ export default function Export() {
 
   return (
     <div>
-      {navigator.clipboard.write && (
+      {navigator.clipboard.write !== undefined && (
```
Comment (katjam) [inline src/components/Export.tsx:9]: classic js :smile:

**Review by katjam — APPROVED** [review]
Nothing here makes me cry.

I'm looking forward to the "more interesting stuff". Pop it through in small PRs if you can.

File: src/components/Export.tsx (hunk 1 above)
Comment (danfishgold) [reply src/components/Export.tsx:9]: you love to see it!

**Comment by danfishgold** [comment]
such high praise

**Pull request merged**